packages/wdio-logger/package.json

```diff
@@ -21,6 +21,8 @@
   "bugs": {
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
+  "types": "./build/node.d.ts",
+  "typeScriptVersion": "3.7.3",
```

Comment:
why is `typeScriptVersion` needed here?

Reply:
it's I think the minimal TS version we support.

Reply:
btw, it's possible to have type definitions for multiple typescript versions!
ex
```
  "typesVersions": {
    "<=3.4": {
      "*": [
        "ts3.4/*"
      ]
    },
    "<=3.6": {
      "*": [
        "ts3.6/*"
      ]
    }
  },
```
just fyi